test/src/test/java/hudson/model/ParametersDefinitionPropertyTest.java

```diff
@@ -29,9 +29,13 @@
 import java.util.logging.Handler;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+
 import net.sf.json.JSONObject;
+
 import org.junit.Test;
+
 import static org.junit.Assert.*;
+
```

Comment:
Beware to not stage whitespaces like this when you're preparing your commit. And as a good practice, review your commit using gitk or something to check that before pushing.

#gsoc